kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/DefaultConfiguration.java

```diff
@@ -535,6 +592,19 @@ public String toString() {
         }
         content.append("  ").append(TRUST_CERTS).append(" = ").append(trustCerts).append(lineSeparator);
 
+        content.append("  ").append(FMP_BUILD).append(" = ").append(fmpBuildEnabled).append(lineSeparator);
+        content.append("  ").append(FMP_BUILD_DISABLE_FOR_MAVEN).append(" = ").append(fmpBuildForMavenDisable).append(lineSeparator);
+        content.append("  ").append(FMP_POM_PATH).append(" = ").append(fmpPomPath).append(lineSeparator);
+        content.append("  ").append(FMP_DEBUG_OUTPUT).append(" = ").append(fmpDebugOutput).append(lineSeparator);
+        content.append("  ").append(FMP_LOGS).append(" = ").append(fmpLogsEnabled).append(lineSeparator);
```

Comment:
We have a lot of identical lines where the only difference is key and value as param. Maybe worth making a method for it?

Reply:
I will make another PR for it.

Reply:
Sure, makes sense.